### PR TITLE
Add initial Access changelog

### PR DESCRIPTION
**Pull request by jroyal**
This needs a pass from @kennyj42 and @ranbel for sure.

### CODE REVIEW

**Review by kennyj42 — APPROVED** [review]

**Comment by kodster28** [comment]
Heyo @jroyal, did you also want a [Markdown page](https://developers.cloudflare.com/style-guide/documentation-content-strategy/content-types/changelog/#markdown-file) for Access specifically?

The current setup would only show Access as part of the top-level `/changelog` page.

**Comment by jroyal** [comment]
@kodster28 yeah if we can.

File: data/changelogs/access.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+link: "/access/changelog/"
+productName: Access
+entries:
+- publish_date: '2024-02-22'
+  title: Access for SaaS OIDC Support
+  description: |-
+    Access for SaaS applications can be setup with OIDC as an authentication method. OIDC and SAML2.0 are now both fully supported.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Access for SaaS applications can be setup with OIDC as an authentication method. OIDC and SAML 2.0 are now both fully supported.
```

File: data/changelogs/access.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2024-02-22'
+  title: WARP as an Identity Authentication Source
+  description: |-
+    Allow users to log in with their WARP session identity. Users need to reauthenticate based on default session durations. WARP authentication identity must be turned on in your device enrollment permissions and can be enabled on a per application basis.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Allow users to log in to Access applications with their WARP session identity. Users need to reauthenticate based on default session durations. WARP authentication identity must be turned on in your device enrollment permissions and can be enabled on a per application basis.
```

File: data/changelogs/access.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+link: "/access/changelog/"
+productName: Access
+entries:
+- publish_date: '2024-02-22'
+  title: Access for SaaS OIDC Support
+  description: |-
+    Access for SaaS applications can be setup with OIDC as an authentication method. OIDC and SAML2.0 are now both fully supported.
+- publish_date: '2024-02-22'
+  title: WARP as an Identity Authentication Source
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
  title: WARP as an identity source for Access
```

File: data/changelogs/access.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  description: |-
+    Allow users to log in with their WARP session identity. Users need to reauthenticate based on default session durations. WARP authentication identity must be turned on in your device enrollment permissions and can be enabled on a per application basis.
+- publish_date: '2023-12-20'
+  title: Unique entity ids in Access for SaaS
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
  title: Unique Entity IDs in Access for SaaS
```

File: data/changelogs/access.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  description: |-
+    All new Access for SaaS applications have unique Entity IDs. This allows for multiple integrations with the same SaaS provide if required. The unique Entity ID has the application audience tag appended. Existing apps are unchanged.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    All new Access for SaaS applications have unique Entity IDs. This allows for multiple integrations with the same SaaS provider if required. The unique Entity ID has the application audience tag appended. Existing apps are unchanged.
```

File: data/changelogs/access.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-12-15'
+  title: Default relay state support in Access for SaaS
+  description: |-
+    Allows Access admins to set a default relay state on Access for SaaS apps
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Allows Access admins to set a default relay state on Access for SaaS apps.
```

File: data/changelogs/access.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-15'
+  title: App launcher supports tags and filters
+  description: |-
+    Access admins can now tag applications and allow users to filter by those tags in the app launcher
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Access admins can now tag applications and allow users to filter by those tags in the App Launcher.
```

File: data/changelogs/access.yaml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-15'
+  title: App launcher customization
+  description: |-
+    Allow Access admins to configure the App Launcher page within the Zero Trust Dashboard
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Allow Access admins to configure the App Launcher page within Zero Trust.
```

File: data/changelogs/access.yaml [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-15'
+  title: View active Access user identities in the dashboard and API
+  description: |-
+    Access admins can now view the full contents of a user's identity and device information at the time of their last session authentication in the Cloudflare dashboard and for all active application sessions.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Access admins can now view the full contents of a user's identity and device information for all active application sessions.
```

File: data/changelogs/access.yaml [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-08'
+  title: Custom OIDC claims for named IDPs
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
  title: Custom OIDC claims for named IdPs
```

File: data/changelogs/access.yaml [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+  description: |-
+    Access admins can add custom claims to the existing named IDP providers. Previously this was locked to the generic OIDC provider.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Access admins can now add custom claims to the existing named IdP providers. Previously this was locked to the generic OIDC provider.
```

File: data/changelogs/access.yaml [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-08-02'
+  title: Azure AD Authentication Context Integration
+  description: |-
+    Support AzureAD authentication contexts directly in Access policies.
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Support Azure AD authentication contexts directly in Access policies.
```

File: data/changelogs/access.yaml [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-15'
+  title: View active Access user identities in the dashboard and API
+  description: |-
+    Access admins can now view the full contents of a user's identity and device information at the time of their last session authentication in the Cloudflare dashboard and for all active application sessions.
+- publish_date: '2023-09-08'
+  title: Custom OIDC claims for named IDPs
+  description: |-
+    Access admins can add custom claims to the existing named IDP providers. Previously this was locked to the generic OIDC provider.
+- publish_date: '2023-08-02'
+  title: Azure AD Authentication Context Integration
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
  title: Azure AD authentication contexts
```

File: data/changelogs/access.yaml [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+  description: |-
+    Support AzureAD authentication contexts directly in Access policies.
+- publish_date: '2023-06-23'
+  title: Access custom pages - deny page
+  description: |-
+    Allow Access admins to customize the deny pages presented by Access to end users
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
    Allow Access admins to customize the block pages presented by Access to end users.
```

File: data/changelogs/access.yaml [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+- publish_date: '2023-09-08'
+  title: Custom OIDC claims for named IDPs
+  description: |-
+    Access admins can add custom claims to the existing named IDP providers. Previously this was locked to the generic OIDC provider.
+- publish_date: '2023-08-02'
+  title: Azure AD Authentication Context Integration
+  description: |-
+    Support AzureAD authentication contexts directly in Access policies.
+- publish_date: '2023-06-23'
+  title: Access custom pages - deny page
```
Comment (ranbel) [inline data/changelogs/access.yaml]:
```suggestion
  title: Custom block pages for Access applications
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**